docs: Add installation instructions for asdf and mise package managers

morihaya opened the pull request:
Add Package Manager Installation Instructions for asdf and mise.

# Description

This PR adds installation instructions for users of the asdf and mise package managers. These additions will help users who prefer these package management systems to easily install and manage kubecolor. By including both asdf and mise, we cater to users of traditional and newer package management tools, improving the accessibility of kubecolor for a wider range of users.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## What you changed

- Added instructions for installing kubecolor using asdf
- Added instructions for installing kubecolor using mise (formerly rtx)

## Why you think we should change it

This change improves the documentation by providing installation instructions for both asdf and mise package managers. It helps users easily install and manage kubecolor, catering to a wider audience with different preferences.

## Related issue (if exists)

morihaya committed:
docs: Add installation instructions for asdf and mise package managers

Add Package Manager Installation Instructions for asdf and mise

This PR adds installation instructions for users of the asdf and mise package managers. These additions will help users who prefer these package management systems to easily install and manage kubecolor.

Changes:
- Add instructions for installing kubecolor using asdf
- Add instructions for installing kubecolor using mise (formerly rtx)

These additions will improve the accessibility of kubecolor for users who rely on these popular package managers, making it easier for them to integrate kubecolor into their workflow. By including both asdf and mise, we cater to users of traditional and newer package management tools.

applejag commented on src/content/docs/setup/install.mdx:
````diff
@@ -104,6 +104,24 @@ Running it via Docker can be nice for just trying out kubecolor:
+asdf global kubecolor latest
+```
+
+== mise
````
Please use the same header style as the rest of the file


```suggestion
## mise
```
same for the asdf header

morihaya replied:
Thank you for the review! I made a simple mistake, but it has been fixed.

applejag commented on src/content/docs/setup/install.mdx:
````diff
@@ -104,6 +104,24 @@ Running it via Docker can be nice for just trying out kubecolor:
+```sh
+mise use kubecolor
+```
+
````
Please move these two to above the "Download binary" header so they're next to the other package headers

morihaya replied:
You're right about your observation. I have corrected the location for the additional information.

applejag commented on src/content/docs/setup/install.mdx:
````diff
@@ -104,6 +104,24 @@ Running it via Docker can be nice for just trying out kubecolor:
+For users using the [asdf](https://github.com/asdf-vm/asdf) package manager.
+
+```sh
+asdf plugin-add kubecolor https://github.com/dex4er/asdf-kubecolor.git
+asdf install kubecolor latest
+asdf global kubecolor latest
````
Kubecolor seems to be available in the shortname index: https://github.com/asdf-vm/asdf-plugins

So this could probably be reduced to:


```suggestion
asdf install kubecolor latest
```

morihaya replied:
You're absolutely right; asdf can indeed be installed with a simpler command. I've made the corrections. Thank you!

applejag reviewed (changes requested):
Very nice! Thanks for this PR!

I had some small comments

morihaya committed:
Accept review feedback from PR

- Change header from == to ##
- Move the position above the "Download binary" header
- Simplify the asdf command

applejag reviewed (approved):
Sweet! Thank you :)
